snuba/datasets/cdc/groupedmessage.py

```diff
@@ -0,0 +1,58 @@
+from snuba.clickhouse import ColumnSet, DateTime, Nullable, String, UInt
+from snuba.datasets import DataSet
+from snuba.datasets.cdc.groupedmessage_processor import GroupedMessageProcessor
+from snuba.datasets.schema import ReplacingMergeTreeSchema
+
+
+class GroupedMessageDataSet(DataSet):
+    """
+    This is a full clone of postgres groupedmessage
+    generated through change capture.
+    """
+
+    def __init__(self):
+        columns = ColumnSet([
+            # columns to maintain the dataset
+            ('commit_id', UInt(64)),
+            ('deleted', UInt(8)),
+            # PG columns
```

Comment:
I wonder whether we need to bring all the columns in clickhouse. We could restrict to the bare minimum set needed to avoid PG in the search backend. @JTCunning what do you think?

Reply:
>We could restrict to the bare minimum set needed to avoid PG in the search backend.

We should restrict to the bare minimum set needed to avoid PG in the search backend.

Reply:
ok, will do

Reply:
The bare minimum is this one in the last version.